src/net.cpp

```diff
@@ -1258,6 +1260,7 @@ bool CConnman::AddConnection(const std::string& address, ConnectionType conn_typ
     switch (conn_type) {
     case ConnectionType::INBOUND:
     case ConnectionType::MANUAL:
+    case ConnectionType::MANUAL_BLOCK_RELAY:
```

Comment:
Addconnection rpc should not support manual block relay, so seems here need not check?

Reply:
It's true that it doesn't support this, just as it doesn't support INBOUND or MANUAL. But leaving it out her would result in a compiler warning for the switch, see comment below:
`// no default case, so the compiler can warn about missing cases`